README.md

```diff
@@ -188,9 +188,9 @@ The main configuration file for the operation of ORT. This configuration is main
 the ORT instance. In contrast to the configuration files in the following, this file rarely changes once ORT is
 operational.
 
-| Format | Scope  | Default location           |
-|--------|--------|----------------------------|
-| HOCON  | Global | `$ORT_CONFIG_DIR/ort.conf` |
+| Format | Scope  | Default location             |
```

Comment:
> The new name "config.yml" is chosen over "ort.yml" to avoid confusion with the repository configuration file ".ort.yml".

Actually, one could also argue that we *do* want to name these files similarly, as they may in parts contain the same configuration options... I just wanted to bring this up to, I'm curious about other opinions from @oss-review-toolkit/core-devs.